doc/source/ray-contribute/docs.ipynb

```diff
@@ -148,7 +148,7 @@
     "```\n",
     "\n",
     "The above snippet was taken from the\n",
-    "[Tune API documentation](https://github.com/ray-project/ray/blob/master/doc/source/tune/api_docs/integration.rst),\n",
+    "[Tune API documentation](https://github.com/ray-project/ray/blob/master/doc/source/tune/api/integration.rst),\n",
```

Comment:
This is no longer the case -- we can remove this line. I'll create an issue about adding the new way of adding API references using `autosummary`.

Reply:
What is no longer the case? Feel free to edit this PR if you think there is a superior fix! 🙂

Reply:
The snippet it's mentioning is outdated now and doesn't match the github link anymore. We can just remove this sentence for now.